doc/examples/Lotka_Volterra_Model_Editor/Lotka_Volterra_Model_Editor.ipynb

```diff
@@ -0,0 +1,206 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from yaml2sbml import YamlModel\n",
+    "\n",
+    "# generate model\n",
+    "model = YamlModel()\n",
+    "\n",
+    "# add ODEs\n",
+    "model.add_ode(state_id='x_1', \n",
+    "              right_hand_side='alpha*x_1 - beta*x_1*x_2', \n",
+    "              initial_value=2)\n",
+    "model.add_ode(state_id='x_2', \n",
+    "              right_hand_side='delta*x_1*x_2 - gamma*x_2', \n",
+    "              initial_value=2)\n",
+    "\n",
+    "# add parameters\n",
+    "model.add_parameter(parameter_id='alpha', nominal_value=2)\n",
+    "model.add_parameter(parameter_id='beta', nominal_value=4)\n",
+    "model.add_parameter(parameter_id='gamma', nominal_value=3)\n",
+    "model.add_parameter(parameter_id='delta', nominal_value=3)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`yaml2sbml` allows to export the `model` object either to YAML or to SBML directly, via  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# write to YAML\n",
+    "model.write_to_yaml('Lotka_Volterra_basic.yaml', over_write=True)\n",
+    "\n",
+    "# write to SBML\n",
+    "model.write_to_sbml('Lotka_Volterra_basic.xml', over_write=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are further functions to: \n",
+    "* get all `parameter_id`s via `model.get_parameter_ids()` \n",
+    "* get a parameter by its id (`model.get_parameter_by_id('alpha')`) \n",
+    "* delete parameters by its id (`model.delete_parameter('alpha')`)\n",
+    "\n",
+    "Similar functions also exist for the other model components."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parameter Estimation Problem\n",
+    "\n",
+    "Now we want to extend the current model \n",
+    "Now we want to extend the current `model` to include all the necessary information for parameter estimation in PEtab. Therefore we load the model from the `.yaml` file and modify the parameters, such that it also contain all information, that is going to be written into the PEtab parameter table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = YamlModel.load_from_yaml('Lotka_Volterra_basic.yaml')\n",
+    "\n",
+    "# extend parameters \n",
+    "model.add_parameter(parameter_id='alpha',\n",
+    "                    nominal_value=2,\n",
+    "                    parameter_scale='log10',\n",
+    "                    lower_bound=0.1,\n",
+    "                    upper_bound=10,\n",
+    "                    estimate=1, \n",
+    "                    over_write=True)\n",
```

Comment:
As overwrite is a word and most/all programs I've seen use `overwrite`, not `over_write`, can change `over_write`->`overwrite` everywhere.